docs/dev/uap/key_concepts.rst

```diff
@@ -0,0 +1,106 @@
+Key Concepts
+============
+
+This document is based on `preliminary discussions in our Google docs <https://docs.google.com/document/d/14H3Rc922-IYy6DIEV76_3hXCQPwl6OmKecJkAheTkI4/edit#heading=h.tthq28g2awwq>`_.
+It describes the key concepts found in the core auth module, and how they touch on other key concepts from other
+modules.
+
+**Kolibri** is a platform with two major purposes:
+#. Serve educational content in "disconnected" scenarios, and support students and teachers with self-paced,
+peer-interactive, and blended learning practices.
+#. Collect information about users and their interaction with that content, and allow it to be synced to other devices
+or to a central repository.
+
+The Kolibri platform has two major software components. The **Distributed Application** (which may be referred to as
+"Kolibri" or the "App") is a software application which serves content (like videos and exercises), stores user data
+to a local database. A Distributed Application is installed and runs on a **Device** (a physical machine such as a
+laptop or tablet). The Distributed Application can store data for one or more **Facilities** (corresponding to a
+physically colocated educational ecosystem, such as a school), and data for a Facility (or a subset of the data from
+a Facility) can be configured to sync to other Devices through a local network, which may have no, limited, or
+intermittent internet access. This could be the LAN at a rural school, or a Raspberry Pi configured to be a wifi
+hotspot that someone carries to refugee camps. More details on the Distributed Application and user data can be found
+below.
+
+A brief foray into the Central Server
+-------------------------------------
+The other major component of the platform is the **Central Server** (also referred to as the "Hub" or "Data Hub"), a
```

Comment:
I will say also that the designations for collections (Facility, Classroom, Learner Group) came from the idea of hierarchical permissions, instead of the other way around. Our interviews with uses (led by @jtamiace) indicated that they wanted permissions like those -- but as a KA Lite user yourself and core developer, your input is also appreciated!